### ISSUE TITLE
The ability to, through an admin panel, enable/disable features of the platform applied per school

### ISSUE THREAD

**Issue by gravitz** [UstadMobile/UstadMobile#723]
The ability by education authority officials to choose (enable/disable) features they want to use in particular schools according to their needs and available facilities. This requirement can be traced back to the inception report

**Comment by mikedawson** [comment]
I don't think this was in the inception report (searching for the word enable, disable does not find it). This isn't in any existing milestone. Generally, if a feature is not wanted in a particular school, then they don't have to use it.
